packages/headless/.eslintrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  extends: ['../../.eslintrc.js'],
+  overrides: [
+    {
+      files: ['*.test.ts'],
+      rules: {
+        '@typescript-eslint/no-explicit-any': 'off',
+        '@typescript-eslint/naming-convention': 'off',
+      },
```

Comment:
If you are talking about the `any`, we're using it everywhere, since this was the new package, I've added this restriction to the code, so it will show the TS error if `any` is used in this package. 
But I think that in the tests we're fine with abusing this rule, cause the correct types will be still forced in the code.